### ISSUE TITLE
Weekend Assignment 6 - Fuhgeddit

### ISSUE THREAD

**Issue by kingcons** [TIY-ATL-ROR-2015-Sep/assignments#396]
## Description

Work in groups to build a simple reddit-like link sharing site. Full description [here](https://github.com/TIY-ATL-ROR-2015-Sep/lectures/blob/master/wk06-rails/wed/questions.md).
### Groups
- Dan, Brian
- Charles, Harrison
- Robert, Darryl
- Carl, Ryan
- Terri, Don
- Sepehr, Danny
- Mans, Joe
## Submission

Post a link to the _main repo_ for the project and mention which branches/pull requests you worked on in the comments.

**Comment by violetaria** [comment]
https://github.com/violetaria/regreddit

well great, i deleted branches on github soooooo

we decided on the routes together

i worked on the following features which have been merged:
1. user registration
2. comments model, controller, views, migration
3. some layout
4. delete post and comment feature
5. comments pagination

I started to use bootstrap to style stuff but did not merge it into the master branch, I'll probably continue to work on it this week as it seems really powerful!
